tools/scripts/docker-compose-up-spanner.sh

```
@@ -0,0 +1,42 @@
+  set +a
+fi
+
+PROJECT="${COMPOSE_PROJECT:-cqrs-es-example-js}"
+DATABASES="-f ../docker-compose/docker-compose-databases.yml"
+SPANNER="-f ../docker-compose/docker-compose-spanner.yml"
+
+# Only the database services (MySQL + migration) are needed from the databases
+# compose for the Spanner path; the read model lives in the same MySQL.
+docker compose -p "${PROJECT}" ${DATABASES} ${SPANNER} down -v --remove-orphans
+
+docker compose -p "${PROJECT}" ${DATABASES} up --remove-orphans -d mysql migration
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)